src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/Utilities/BaseTest.cs

```diff
@@ -564,6 +564,32 @@ protected bool RunCommand (string command, string arguments)
 			}
 		}
 
+		protected bool IgnoreUnsupportedConfiguration (AndroidRuntime runtime, bool aot = false, bool release = false)
+		{
+			if (runtime == AndroidRuntime.NativeAOT) {
+				// NativeAOT is release-only, AOT is always implied
+				if (release) {
+					return false;
+				}
+
+				Assert.Ignore ($"NativeAOT: unsupported configuration (release == {release})");
```

Comment:
I believe the way NUnit works, `Assert.Ignore()` throws an exception that tells NUnit to skip.

So, I think you could make this method void and you don't need an `if-return` statement in each test.